server/src/main/kotlin/com/mobilejazz/colloc/feature/decoder/CsvLocalizationDecoder.kt

```diff
@@ -0,0 +1,58 @@
+package com.mobilejazz.colloc.feature.decoder
+
+import com.mobilejazz.colloc.domain.model.Dictionary
+import com.mobilejazz.colloc.domain.model.Language
+import com.opencsv.CSVReader
+import java.io.FileReader
+import java.nio.charset.StandardCharsets
+
+typealias DictionaryEditor = MutableMap<Language, MutableMap<String, String>>
```

Comment:
Maybe better to have it in the `Dictionary.kt` file with the `Dictionary` typealias. 

Also, I would follow kotlin naming standard to have mutable objects and I would call it `MutableDictionary`.

Reply:
I think it's a nice addition for the future to keep it public in the`Dictionary.kt`